docs/sources/setup/install/helm/deployment-guides/aws.md

```diff
@@ -0,0 +1,763 @@
+
+1. Create a bucket policy file named `bucket-policy-chunk.json` with the following content:
+
+    ```json
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Sid": "Statement1",
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "arn:aws:iam::<account id>:role/LokiServiceAccountRole"
+                },
+                "Action": [
+                    "s3:PutObject",
+                    "s3:GetObject",
+                    "s3:DeleteObject",
+                    "s3:ListBucket"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::<chunk bucket name>",
+                    "arn:aws:s3:::<chunk bucket name>/*"
+                ]
+            }
+        ]
+    }
+    ```
+    **Make sure to replace the placeholders with your AWS account ID and the bucket name.**
+
+1. Add the policy to the bucket:
+
+    ```bash
+    aws s3api put-bucket-policy --bucket <Your S3 bucket eg. `loki-aws-dev-chunks`> --policy file://bucket-policy-chunk.json
+    ```
+1. Create a bucket policy file named `bucket-policy-ruler.json` with the following content:
+
+    ```json
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Sid": "Statement1",
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "arn:aws:iam::<account id>:role/LokiServiceAccountRole"
+                },
+                "Action": [
+                    "s3:PutObject",
+                    "s3:GetObject",
+                    "s3:DeleteObject",
+                    "s3:ListBucket"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::<ruler bucket name>",
+                    "arn:aws:s3:::<ruler bucket name>/*"
+                ]
+            }
+        ]
+    }
+    ```
+    **Make sure to replace the placeholders with your AWS account ID and the bucket name.**
+
+1. Add the policy to the bucket:
+
+    ```bash
+    aws s3api put-bucket-policy --bucket <Your S3 bucket eg. `loki-aws-dev-ruler`> --policy file://bucket-policy-ruler.json
+    ```
+
+## Deploying the Helm chart
+
+Before we can deploy the Loki Helm chart, we need to add the Grafana chart repository to Helm. This repository contains the Loki Helm chart.
+
+1. Add the Grafana chart repository to Helm:
+
+    ```bash
+    helm repo add grafana https://grafana.github.io/helm-charts
+    ```
+1. Update the chart repository:
+
+    ```bash
+    helm repo update
+    ```
+1. Create a new namespace for Loki:
+
+    ```bash
+    kubectl create namespace loki
+    ```
+### Loki Basic Authentication
+
+Loki by default does not come with any authentication. Since we will be deploying Loki to AWS and exposing the gateway to the internet, we recommend adding atleast basic authentication. In this guide we will give Loki a username and password:
```

Comment:
```suggestion
Loki by default does not come with any authentication. Since we will be deploying Loki to AWS and exposing the gateway to the internet, we recommend adding at least basic authentication. In this guide we will give Loki a username and password:
```